notebook/mortgage_e2e_gquant/mortgage_gquant_plugins.py

```diff
@@ -749,29 +785,34 @@ def mortgage_gquant_run(run_params_dict):
 
 
 def print_ram_usage(worker_name='', logger=None):
+    '''Display host RAM usage on the system using free -m command.'''
     import os
 
     if logger is None:
-        logger = get_mortgage_plugins_logger()
+        logmgr = MortgagePluginsLoggerMgr()
+        logger = logmgr.get_logger()
 
     tot_m, used_m, free_m = \
         map(int, os.popen('free -t -m').readlines()[-1].split()[1:])
     logger.info(
         worker_name + 'HOST RAM (MB) TOTAL {}; USED {}; FREE {}'
         .format(tot_m, used_m, free_m))
 
+    if logger is None:
+        logmgr.cleanup()
```

Comment:
Oops, bug. Logger cleanup will never happen. I'll fix and re-submit.

Reply:
https://github.com/avolkov1/gQuant/blob/CUQ-5-Mortgage-Example-Using-gQuant/notebook/mortgage_e2e_gquant/mortgage_gquant_plugins.py#L803

Fixed in latest commit.